### ISSUE TITLE
import qiskit doesn't work

### ISSUE THREAD

**Issue by quantumjim** [Qiskit/qiskit-terra#1482]
### Informations

- **Qiskit Terra version**: 0.7.0
- **Python version**: 3.7.0
- **Operating system**: OSX

### What is the current behavior?
```
---------------------------------------------------------------------------
ModuleNotFoundError                       Traceback (most recent call last)
<ipython-input-5-d44eb5a7400f> in <module>
----> 1 import qiskit
      2 
      3 q = QuantumRegister(1)
      4 c = ClassicalRegister(1)
      5 circ = QuantumCircuit(q, c)

/usr/local/lib/python3.7/site-packages/qiskit/__init__.py in <module>
     24 from qiskit.circuit import QuantumRegister
     25 from qiskit.circuit import QuantumCircuit
---> 26 from .tools.compiler import (compile, execute)
     27 
     28 # Please note these are global instances, not modules.

/usr/local/lib/python3.7/site-packages/qiskit/tools/__init__.py in <module>
     16 """
     17 
---> 18 from .compiler import (compile, execute)
     19 from .monitor import job_monitor

/usr/local/lib/python3.7/site-packages/qiskit/tools/compiler.py in <module>
     10 import logging
     11 
---> 12 from qiskit import transpiler
     13 from qiskit.transpiler._passmanager import PassManager
     14 from qiskit.converters import circuits_to_qobj

/usr/local/lib/python3.7/site-packages/qiskit/transpiler/__init__.py in <module>
     13 from ._fencedobjs import FencedDAGCircuit, FencedPropertySet
     14 from ._basepasses import AnalysisPass, TransformationPass
---> 15 from ._transpiler import transpile, transpile_dag
     16 from ._parallel import parallel_map
     17 

/usr/local/lib/python3.7/site-packages/qiskit/transpiler/_transpiler.py in <module>
     22 from qiskit.converters import dag_to_circuit
     23 from ._parallel import parallel_map
---> 24 from .passes.mapping.unroller import Unroller
     25 
     26 

/usr/local/lib/python3.7/site-packages/qiskit/transpiler/passes/__init__.py in <module>
     10 from .cx_cancellation import CXCancellation
     11 from .fixed_point import FixedPoint
---> 12 from .mapping.check_map import CheckMap
     13 from .mapping.basic_mapper import BasicMapper
     14 from .mapping.direction_mapper import DirectionMapper

ModuleNotFoundError: No module named 'qiskit.transpiler.passes.mapping'
```

### Steps to reproduce the problem

`import qiskit`, or `from qiskit import *`, etc.

### What is the expected behavior?

Qiskit should get imported.

### Suggested solutions

Problem possibly related to #1455 ?

**Comment by ajavadia** [comment]
This should be fixed by #1480.

**Comment by mtreinish** [comment]
Yeah, this will be fixed by #1480 . Also fwiw I wanted to add this to our ci so we could catch failures like this some time ago with:  https://github.com/Qiskit/qiskit-terra/pull/1201

**Comment by mtreinish** [comment]
Although now that I think about it some more I'm not sure that #1201 will catch this because it's using editable mode which just sets a link for the package.

**Comment by mtreinish** [comment]
This should be fixed now that #1480 has merged.
